_opcodes/op116.html

```diff
@@ -10,6 +10,16 @@
 iwd2: 1
 pst: 1
 ---
-Removes invisibility from the targeted creature(s).<br/>
-This effect ignores durations attributed to it.<br/>
+Removes invisibility from the targeted creature(s).
+
+{% capture note %}
+This effect ignores durations attributed to it.
```

Comment:
Ok, so reverting to the old style? Or maybe changing to "The value should always be set to ..."?